docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +11,16 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-0-1]]
+=== Logstash 6.0.1 Release Notes
+* Elasticsearch Output: Ignore event's type field for the purpose of setting document _type if cluster is es 6.x or above
+* Fix bug where plugins that used nested hashes for configuration would not work correctly
+* Ensure logstash always terminates on second sigint, there were some situations where executing two sigints would not kill Logstash
+* Fix bug where persistent queue could not be opened when in some states
+* Fix an issue where the aggregate filter would not always shut down correctly
+* Fix a large performance regression in the grok filter that has been with us since Logstash 5.x . 
```

Comment:
I think you mean to say, "...see a significant improvement in grok performance..."

Incidentally, this statement seems a little negative to me (akin to saying "we had a big problem and it took us a long time to fix it").

Reply:
Well, that is the case here, I'd rather not sugarcoat it. This is very relevant to users going from 2.x -> 6.0, since they will seen no improvement.